docs/manuscript.Rmd

```diff
@@ -9,7 +9,28 @@
 
 ## Introduction
 
-Scholarly research has been said to face a reproducibility crisis, but the underlying cause of this is the incentive system. 
+Scholarly research faces severe threats to its sustainability has been said to face a reproducibility crisis [@Baker2016] (amongst other pernicious problems), but the underlying cause of this is the way we report and reward research . 
+The current scholarly communication system is organized around researchers who publish papers in journals published (mostly) by for-profit publishers. 
+Many of these journals have a page limit leading to artificial scarcity which subsequently increases the prestige of such a journal. 
+Furthermore, scientific communication is organized around a few publishers which increases the centralization of said communication tremendously [@Lariviere2015]. 
+This introduces knowledge discrimination, as publishers are able to influence who can access scientific knowledge, what gets published, and allows for other single points of failure to arise. 
+In order to have a sustainable scholarly communication- and a more effective research system, severe changes are necessary from our perspective.
+
+The way scholarly communication is designed has large effects on the behavior of individual researchers, universities, and funders. 
```

Comment:
The design of scholarly communication sets the conditions for the behavior of researchers, universities, and funders.